substrate/client/cli/src/error.rs

```diff
@@ -78,6 +80,19 @@ pub enum Error {
 
 	#[error(transparent)]
 	GlobalLoggerError(#[from] sc_tracing::logging::Error),
+
+	#[error(
+		"Starting an authorithy without network key in {0}.
+		\n This is not a safe operation because the old identity still lives in the dht for 36 hours.
+		\n Because of it your node might suffer from not being properly connected to other nodes for validation purposes.
```

Comment:
```suggestion
		\n This is not a safe operation because other authorities in the network may depend on your node having a stable identity.
		\n Otherwise these other authorities may not being able to reach you.
```